docs/source/pages/tutorials/use_verification_summary_attestation.rst

```diff
@@ -0,0 +1,159 @@
+
+This tutorial explains how to use the Verification Summary Attestations (VSA) generated by Macaron, using the VSAs for the `Graal Development Kit (GDK) <https://graal.cloud/gdk/>`_ artifacts as an example.
+
+For more information about VSAs, please refer to the :ref:`Verification Summary Attestation page<vsa>`. To use Macaron to generate VSAs see this :ref:`tutorial <gen-vsa_tutorial>`.
+
+* https://slsa.dev/spec/v1.0/verification_summary
+
+--------
+Use case
+--------
+
+Imagine you are a consumer of GDK artifacts and want to verify whether they are produced by a secure, :term:`SLSA`-compliant build service and sourced from a trusted source code repository. You need to confirm whether a provenance record for this artifact is published and has been verified. The GDK team must keep the details of their build pipeline confidential while still communicating that verification has occurred.
+
+A VSA allows you to assess the security properties of an artifact without needing direct access to the provenance details. This process involves delegating the policy decision to Macaron. Macaron receives the provenance of the build as input, analyzes various aspects of the build, and verifies the gathered data against a Datalog policy. It then generates a VSA that attests to the artifacts produced by the build, which is published alongside the artifacts.
+
+-------
+Example
+-------
+
+GDK is an Oracle build of the open source Micronaut® framework. GDK provides a curated set of Micronaut framework modules which are built from source and published on the `Oracle Maven repository <https://maven.oracle.com/public>`_. If a GDK artifact is verified by Macaron, you should be able to to find a corresponding VSA on Oracle Maven repository. Let's consider, the ``io.micronaut/micronaut-core@4.6.3-oracle-00001`` JAR artifact which is published at `<https://maven.oracle.com/public/io/micronaut/micronaut-core/4.6.3-oracle-00001/micronaut-core-4.6.3-oracle-00001.jar>`_. In order to verify the artifact with Macaron, you can follow the following steps:
+
+''''''''''''''''
+Download the VSA
+''''''''''''''''
+
+Check wether a VSA is published for the artifact and download it for further examination.
+
+.. code-block:: shell
+
+  curl -O https://maven.oracle.com/public/io/micronaut/micronaut-core/4.6.3-oracle-00001/vsa.intoto.jsonl
+
+''''''''''''''''''''''''''''''''''''
+Manual inspection of the VSA content
+''''''''''''''''''''''''''''''''''''
+
+.. code-block:: shell
+
+    cat vsa.intoto.jsonl | jq -r '.payload' | base64 -d | jq
+
+The output of the this command should look like below:
+
+.. toggle::
+
+    .. code-block:: json
+
+        {
+            "_type": "https://in-toto.io/Statement/v1",
+            "subject": [
+            {
+                "uri": "pkg:maven/io.micronaut/micronaut-core@4.6.3-oracle-00001?type=jar",
+                "digest": {
+                "sha256": "67ccb8e1a69da33ac7494b90d296db4ed3b2cdf3ccd267432dd2b7fac9b9bb12"
```

Comment:
nit: I think this line should be indented one more level.

Reply:
See commit [32e104c](https://github.com/oracle/macaron/pull/858/commits/32e104c12ea49dec6675910dc6e9a3ea59baf4d9)